src/reducers/projects.js

```diff
@@ -0,0 +1,149 @@
+const initialStateUser = {
+  allProjects: [], 
+  userProjects: []
+}
+
+const projects = (state = initialStateUser, action, root) => {
+  switch (action.type) {
+    case 'GET_USER_PROJECTS':
+      let beginningDate
+      let dueDate
+      action.projects.map(project => {
+        return (
+          beginningDate = new Date(project.beginningDate),
+          dueDate = new Date(project.dueDate),
+          project.beginningDate = beginningDate.toISOString().slice(0,10),
+          project.dueDate = dueDate.toISOString().slice(0,10)
+        )
+      })
+      const userProjectsTemp = {
+        ...state,
+        userProjects: action.projects
+      };
+      return userProjectsTemp;
+
+    case 'SET_NEW_PROJECT':
+      beginningDate = new Date(action.response.project.beginningDate)
+      dueDate = new Date(action.response.project.dueDate)
+      beginningDate = beginningDate.toISOString().slice(0,10)
+      dueDate = dueDate.toISOString().slice(0,10)
```

Comment:
Create some helper method to have the logic to update the project object with desired date formats or customisations.